.changes/unreleased/Fixes-20220723-215330.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Fixes
+body: CT-700 [Bug] Logging tons of asterisks when sensitive env vars are missing
```

Comment:
Can you please update this message to state what the solution is instead of what the problem was?  Something like `Ignore empty strings passed in as secrets` would be great!

Reply:
No need to run `changie` again.  You can directly edit this yaml file.

Reply:
I have just updated the message in the yaml file and commited to the repo.
Thank you for the review.